train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -51,4 +53,16 @@ public void emergencyBreak() {
 		this.referenceSpeed = 0;
 	}
 
```

Comment:
Infinite loop in the main thread may result in a total failure of the application